packages/webapp/pages/[userId]/index.tsx

```diff
@@ -252,6 +251,22 @@ const ProfilePage = ({ profile }: ProfileLayoutProps): ReactElement => {
               />
             )}
           </ActivityContainer>
+          <ActivityContainer>
+            <ActivitySectionTitle>
+              Top tags by reading days
+            </ActivitySectionTitle>
+            <div className="grid grid-cols-1 tablet:grid-cols-2 gap-3 tablet:gap-10">
+              {readingHistory.userReadingTopTags?.map((topTag) => (
```

Comment:
Maybe just call this "Tag"

Reply:
I think it would kinda look odd if we're gonna do `tag.tag` (like from the key) since it has a property tag. What do you think?

Reply:
Ah yes makes sense 🙏